notebook/static/notebook/less/menubar.less

```diff
@@ -140,3 +140,7 @@ ul#help_menu li a{
 }
 
 //end submenu
+
+.document_details_table {
+    width: 100%;
+}
```

Comment:
I don't think this belongs in `menubar.less`, which is about styling for the menus themselves. I'm not sure what is the right place for it, though.